a11y/a11y.test.js

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import path from 'path'
+
+import puppeteer from 'puppeteer'
+import execa from 'execa'
+
+import { mockServer } from './mock-server'
+
+const OTP_RR_CONFIG_FILE_PATH = './config.yml'
+const OTP_RR_CONFIG_BACKUP_PATH = './config.non-test.yml'
+const OTP_RR_TEST_CONFIG_PATH = './a11y/test-config.yml'
+
+beforeEach(() => {
+  // backup current config file
+  if (fs.existsSync(OTP_RR_CONFIG_FILE_PATH)) {
+    fs.renameSync(
+      OTP_RR_CONFIG_FILE_PATH,
+      OTP_RR_CONFIG_BACKUP_PATH
+    )
+    console.log('Backed up current OTP-RR config file')
+  }
+  // copy over test config file
+  fs.copyFileSync(
+    OTP_RR_TEST_CONFIG_PATH,
+    OTP_RR_CONFIG_FILE_PATH
+  )
+  console.log('Copied a11y test config file')
+
+  // Build OTP-RR main.js using new config file
+  execa.sync('yarn', ['build'])
+  console.log('Built OTP-RR')
+
+  // Launch mock OTP server
+  const MOCK_SERVER_PORT = 9999
+  mockServer.listen(MOCK_SERVER_PORT, () => {
+    console.log(`Mock response server running on http://localhost:${MOCK_SERVER_PORT}`)
+  })
+})
+
+afterEach(() => {
+  fs.unlinkSync(OTP_RR_CONFIG_FILE_PATH)
+  if (fs.existsSync(OTP_RR_CONFIG_BACKUP_PATH)) {
+    fs.renameSync(
+      path.resolve(OTP_RR_CONFIG_BACKUP_PATH),
+      path.resolve(OTP_RR_CONFIG_FILE_PATH)
+    )
+  }
+  console.log('Restored original OTP-RR config file')
```

Comment:
Should the mockServer be shut down too here?

Reply:
Have added a server shut down here but am still had to keep the `--force-exit` to get jest to exit properly.